cypress/integration/login.js

```diff
@@ -9,16 +9,17 @@ describe('Login functionality', function() {
 
     it('Should be able to login with role User', function () {
         cy.get('[id="loginForm.userId"]').click({force:true})
-        cy.get('[aria-label="Demo User"]').click()
+        cy.get('[aria-label="TestCon User 26"]').click()
         cy.get('[id="loginForm.role"]').click({force:true})
-        cy.get('[aria-label="User"]').click()
+        cy.get('[aria-label="Team Lead"]').click()
         cy.get('[type="submit"]').click()
 
         cy.url().should('include', '/time-logging')
         cy.get('.page__title').contains('Timesheets')
         cy.get('.calendar').should('be.visible')
         cy.get('.tile.form').should('be.visible')
-        cy.get('.user-info__title').contains('Demo User')
-        cy.get('.main-nav').find('li').should('have.length', 1)
+        cy.get('.user-info__title').contains('TestCon User 26')
+        cy.get('.main-nav').find('li').should('have.length', 2)
+        cy.get('.calendar--selected').should('have.class', 'calendar--today')
```

Comment:
We should assert the actual today's date value, e.g. today is 13th day, tomorrow it will be 14th.

Reply:
Made changes